### ISSUE TITLE
ATmega328p-MMH only one time programming after that showing chip enable program error 

### ISSUE THREAD

**Issue by Reshma657** [arduino/Arduino#9031]
Case Description: 
We are using ATmega328P-MMH chip for the project. we have made 2 boards. when we are programming the Chip using the progIsp only once we are successful to load the code but after that it showing "program enable chip error". It happened on both boards. Please find the fuse bit information in attachment and also schematic for the IC. I am using internal oscillator. 
_please_ note that Arduino uno board I have tried same code it is working on uno board. 
Requesting you to kindly provide help ASAP. 
![image](https://user-images.githubusercontent.com/42932145/60409265-8225b680-9be0-11e9-8277-048cd39eed50.png)

**Comment by per1234** [comment]
This issue tracker is only to be used to report bugs or feature requests for Arduino's software and documentation. This topic is more appropriate for the Arduino Forum. I'm sure we'll be able to help you with your problem over there.

Please do this:
1. Read http://forum.arduino.cc/index.php?topic=148850
2. If you haven't already done so, create a new thread in the appropriate section of the forum (http://forum.arduino.cc/), following all the rules. Be sure to post all the information we might need to help you.
